### PR TITLE
Shim to resolve conflict with IE11 due to usage of .includes()

### PR DESCRIPTION
**Pull request by Ricokola**
Related to: #9482 

The usages of `.includes()` instead of `.match()` has created conflicts with IE11 and therefore, some plugins have stop working. A shim has been created to fix the issues encounter in IE11.

### CODE REVIEW

File: src/shims/includes.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/**
+ * String.prototype.includes() polyfill
+ * @author Ricokola
+ * @license MIT
+ */
+if ( !String.prototype.includes ) {
+	String.prototype.includes = function( string ) {
+		console.log( string );
```
Comment (duboisp) [inline src/shims/includes.js:8]: Please remove the console.log

**Review by duboisp — CHANGES_REQUESTED** [review]

**Comment by EricDunsworth** [comment]
@duboisp Regarding this:
> Note: The IE11 detection is made through the user agent detection and not on the availability of the "includes" function on String object.

It's true that IE11 detection is performed by checking its user agent string, but that check currently uses ``includes()``. So this shim is needed to avoid a JS console error when the check runs.

**PS:**
The check used to use ``match()`` in the past:
https://github.com/wet-boew/wet-boew/blob/d93b46e8b56dd50009b8192705d004867298f5ff/src/core/wb.js#L258

But #9482 recently changed it to use ``includes()``:
https://github.com/wet-boew/wet-boew/blob/999b5e021718a8fe945a8b4054333c9b62de3722/src/core/wb.js#L258

**Comment by duboisp** [comment]
fyi - Test page: http://universallabs.org/wet/labs/pr9505/

**Review by duboisp — APPROVED** [review]
Tested locally in my old IE11 and work as expected. Now what was working continu to work as expected. ex. expand/collapse.

**Pull request merged**